_posts/2020-07-01-openzaak-market-consultation-workshops.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "The OpenZaak market consultation workshops"
+author: Felix Faassen
+type: blogpost
+category: codebase stewardship
+excerpt: how to build a financially sustainable open source codebase ecosystem from scratch
+---
+
+# The OpenZaak market consultation workshops
+
+![governance workshop participants do the wave](/assets/openzaak-community-wave.gif)
+
+I started working as a Foundation for Public Code codebase steward at the start of May 2020. My first big assignment was to support a coalition of Dutch municipalities with organizing and facilitating a series of workshops as part of the [OpenZaak market consultation](https://github.com/open-zaak/open-zaak-market-consultation). This post describes my experience with the workshop process. It doesn't cover the workshop outputs, which will be published in the market consultation report later this month.
+
+## Background: the OpenZaak codebase
+
+OpenZaak is an open-source case register platform for municipalites which implements the [nationwide case management APIs (zaakgericht werken APIs)](https://www.vngrealisatie.nl/producten/api-standaarden-zaakgericht-werken). These establish a standard modern application architecture, making it easy for municipalities to create and deliver new online services to citizens and share data with other municipalities.
+
+OpenZaak was commissioned from a small developer by a group of more than 40 Dutch municipalities working together, including [Dimpact](https://www.dimpact.nl/), an association for shared municipal technology procurement. By February 2020, the municipalities were ready to take OpenZaak into production and further develop the software together with different software vendors. Amsterdam, Haarlem, Utrecht and the Dimpact municipalities asked the Foundation for Public Code to help them answer this question: how can we create a diverse and open community of market parties around OpenZaak
```

Comment:
Was the developer small? Or the vendor?

Reply:
good point I removed small, I feel more comfortable with that.